patches/server/0005-Tuinity-Server-Config.patch

```diff
@@ -153,23 +162,19 @@ index 0000000000000000000000000000000000000000..89a4567a8ee9591bab78c444f45cf733
 +        }
 +
 +        private boolean getBoolean(final String path, final boolean dfl) {
-+            this.config.addDefault(path, Boolean.valueOf(dfl));
-+            return this.config.getBoolean(path, dfl);
++            return this.config.getBoolean(path, TuinityConfig.worldConfigDefault.getBoolean(path, dfl));
 +        }
 +
 +        private int getInt(final String path, final int dfl) {
-+            this.config.addDefault(path, Integer.valueOf(dfl));
-+            return this.config.getInt(path, dfl);
++            return this.config.getInt(path, TuinityConfig.worldConfigDefault.getInt(path, dfl));
 +        }
 +
 +        private double getDouble(final String path, final double dfl) {
-+            this.config.addDefault(path, Double.valueOf(dfl));
```

Comment:
defaults still need to be added

Reply:
Global default values will be meaningless if default values are added for each world

Reply:
default values need to be added to the default section, as of right now no defaults are added - thus values aren't automatically added to config

Reply:
they will be added to `defaults` when `TuinityConfig.worldConfigDefault.getDouble(path, dfl)`

Reply:
fixed